README.md

```diff
@@ -4,7 +4,8 @@ The API Insights OpenAPI ruleset is a project to programatically check conforman
 
 This module implements a set of lintable REST API best practices, linters can be used validate API Specs in Swagger2 or OAS3 format. The project implements 2 linter rulesets. 
 1. REST Guidelines :  Lintable REST API best practices related to API Design, API implementation, API testing and developer experience. For details refer [rest-guidelines.md](docs/rest-guidelines.md)
-2. Doc Completeness: set of rules provided to ensure the end user document generated from API specs are of good quality. It includes rules and examples are included for every schema, description provided for every attribute, etc. For details refer to [doc-completeness.md](docs/doc-completeness.md)
+2. Contract: set of rules provided to ensure the end user document generated from API specs are of good quality. For details refer to [doc-completeness.md](docs/doc-completeness.md)
```

Comment:
Here I reused the doc-completeness.md for reference, since things are all covered in that doc. If we split the markdown doc, so we need to update all of them in docs and also the documentation repo. Should we do this?

Reply:
And I dont know if next time, they think completeness will be splited in another manner, so we need to update doc split again. This is another reason I keep it as it was.